### DISCUSSION TITLE
Got 50 MB JSON, and bun run --watch serverwh.ts is killed in 1GB VPS

### DISCUSSION THREAD

**Discussion by bitdom8** [oven-sh/bun#2708] [General]
Got 50 MB JSON, and `bun run --watch serverwh.ts` is killed in 1GB VPS.

```
import bforex from "./bforex.json";
import bcrypto from "./bcrypto.json";
import bstocks from "./bstocks.json";
```

The last one is pushing the process to end

**Comment by Jarred-Sumner** [comment]
It's probably an OOM.

Does it happen without `--watch`?

**Comment by bitdom8** [comment]
Yes, it happens in Linux with my 1GB VPS with/w/o --watch, however, fine with my Windows tech-savvy computer
